exercises/high-scores/.meta/src/reference/java/HighScores.java

```diff
@@ -0,0 +1,32 @@
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.OptionalInt;
+
+import static java.util.Arrays.copyOf;
+import static java.util.Arrays.sort;
+
+class HighScores {
+    private int[] highScores;
+    public HighScores(int[] highScores) {
+        this.highScores = highScores;
+    }
+
+    int[] scores() {
+        return highScores;
+    }
+
+    int latest() {
+        return highScores[highScores.length - 1];
+    }
+
+    int personalBest() {
+        OptionalInt personalBest = Arrays.stream(highScores).max();
+        return personalBest.orElse(0);
+    }
+
+    Integer[] personalTopThree() {
```

Comment:
```suggestion
    int[] personalTopThree() {
```

Everything else uses primitives - so why would we return a boxed array?